packages/guides/src/Nodes/Menu/ParsedMenuEntryNode.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace phpDocumentor\Guides\Nodes\Menu;
+
+use phpDocumentor\Guides\Nodes\AbstractNode;
+use Stringable;
+
+/**
+ * This node contains the result of parsing the menu entries for a menu.
+ *
+ * @extends AbstractNode<String>
+ */
+final class ParsedMenuEntryNode extends AbstractNode implements Stringable
```

Comment:
The name of this class is not really describing what kind of node this is, Would `TitledMenuEntryNode` cover the purpose?

Reply:
The thing is  a Menu entry usually contains the resolved links of the Menu. So I am not really happy with also calling it MenuEntry. But I also have no better Idea...

Reply:
How about MenuDefinitionLineNode?